tests/functions_tests/test_arrays.py

```diff
@@ -37,6 +37,12 @@
      'args': {'pad_width': ((0, 0), (0, 0), (2, 2), (2, 2)),
               'mode': 'edge'},
      'name': 'pad_edge'},
+    {'ops': 'pad', 'input_shape': (1, 2, 3, 4),
+     'input_argname': 'x',
+     'args': {'pad_width': ((0, 0), (0, 0), (2, 2), (2, 2)),
+              'mode': 'constant',
+              'constant_values': -1},
```

Comment:
Not your fault, but could you test a random float (e.g., `constant_values=4.2`) as well?

Reply:
It seems like `chainer.functions.pad` accepts only integer type for `constant_values`.